x-pack/plugins/infra/server/lib/domains/log_entries_domain/builtin_rules/filebeat_system.ts

```diff
@@ -20,6 +29,15 @@ export const filebeatSystemRules = [
       exists: ['system.auth.message'],
     },
     format: [
+      {
+        constant: '[Syslog][auth] ',
```

Comment:
Was this meant to be

```suggestion
        constant: '[System][auth] ',
```

?